### DISCUSSION TITLE
Custom hypothesis client

### DISCUSSION THREAD

**Discussion by galachad** [quarto-dev/quarto-cli#4837] [Feature Requests]
Currently, it's not possible to define a custom js hypothesis client.

It should be possible to set the `https://hypothes.is/embed.js` to a custom url e.g. https://custom-hypothesis-client.internal/boot.js

https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/html/hypothesis/hypothesis.ejs#L6 

Thank you for implementing this.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/4840
